### PR TITLE
Feature/cards

### PR DESCRIPTION
**Pull request by justsealan**
## Name
Create the cards section and style the sign-up form.

## Description
Added a carousel for the cards section. Also had to rewrite the sign-up form as it broke down after adding the carousel.

## Related to
Card carousel user story.
(https://github.com/Haseeb3927/Booster/projects/1#card-79273563)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have carefully reviewed my own code
- [x] I have commented my code
- [x] I have updated any documentation

### CODE REVIEW

File: client/src/components/CardsCarousel/CardCarousel.js [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+		id: 7,
+		title: "Energiser 7",
+		description: "Energiser 7 description",
+		image: "https://source.unsplash.com/random/400x400",
+		link: "https://www.google.com",
+		instructions: "Lorem ipsum",
+		likes: "100",
+		dislikes: "10",
+	},
+];
```
Comment (Haseeb3927) [inline client/src/components/CardsCarousel/CardCarousel.js:90]: That's a good practice Selahaddin. But our database is connected. So we don't need this hard coded data.

File: client/src/App.js [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
-import Home from "./pages/mainPage/components/home/Home.js";
+// import About from "./pages/About";
+import Home from "./pages/Home";
 
 const App = () => (
 	<Routes>
+
 		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
+		{/* <Route path="/about/this/site" element={<About />} /> */}
```
Comment (Haseeb3927) [inline client/src/App.js:10]: Please delete the commented out code if no longer needed.

File: client/src/App.js [hunk 3]
```diff
@@ -1,12 +1,13 @@
 import { Route, Routes } from "react-router-dom";
 
-import About from "./pages/secondPage/About";
-import Home from "./pages/mainPage/components/home/Home.js";
+// import About from "./pages/About";
```
Comment (Haseeb3927) [inline client/src/App.js:3]: Please delete the commented out code if no longer needed.

File: client/src/components/Jumbotron/Registration.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+	};
+
+	const handleMouseDownPassword = (event) => {
+		event.preventDefault();
+	};
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		const data = new FormData(event.currentTarget);
+		// eslint-disable-next-line no-console
```
Comment (Haseeb3927) [inline client/src/components/Jumbotron/Registration.js:46]: Delete this comment please.

File: client/src/components/Theme.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+			main: "#F41A0C",
+			contrastText: "#DDEE14",
+		},
+	},
+	typography: {
+		fontFamily: "Electrolize",
+	},
+});
+
+export default Theme;
```
Comment (Haseeb3927) [inline client/src/components/Theme.js:23]: That's a good practice you created your own theme.

File: client/src/App.js (hunk 3 above)
Comment (justsealan) [reply client/src/App.js:3]: comments removed

File: client/src/components/Jumbotron/Registration.js (hunk 4 above)
Comment (justsealan) [reply client/src/components/Jumbotron/Registration.js:46]: comments removed

**Review by Haseeb3927 — APPROVED** [review]
Looks good to me

**Pull request merged**